.circleci/config.yml

```diff
@@ -221,7 +221,7 @@ jobs:
     run_tests_custom_tokenizers:
         working_directory: ~/transformers
         docker:
-            - image: circleci/python:3.6
+            - image: circleci/python:3.7
```

Comment:
Do you dislike python 3.6?

Reply:
I figured python3.7 is probably more used now, but I can revert !